<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-DAPHEM15" failures="1" tests="8" name="happyfox Test2" time="83.94" errors="0" timestamp="2020-11-10T03:18:25 IST">
  <testcase classname="happyFox_TestCases.Test_Scenario_1" name="a_login" time="5.044"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_1" name="b_creatingStatus" time="16.289"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_2" name="a_CreateNewTicket" time="6.166">
    <failure type="java.lang.AssertionError" message="expected [New Ticket  - Tenmiles - powered by HappyFox] but found [New Ticket - Tenmiles - powered by HappyFox]">
      <![CDATA[java.lang.AssertionError: expected [New Ticket  - Tenmiles - powered by HappyFox] but found [New Ticket - Tenmiles - powered by HappyFox]
at happyFox_utilities.Assertion_helper.verifyTitle(Assertion_helper.java:22)
at happyFox_TestCases.Test_Scenario_2.a_CreateNewTicket(Test_Scenario_2.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- a_CreateNewTicket -->
  <testcase classname="happyFox_TestCases.Test_Scenario_2" name="b_login" time="3.25"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_2" name="c_SelectTicket_responding" time="13.349"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_3" name="a_login" time="3.878"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_3" name="b_deleteStatus" time="17.548"/>
  <testcase classname="happyFox_TestCases.Test_Scenario_3" name="c_logout" time="0.123"/>
</testsuite> <!-- happyfox Test2 -->
